### ISSUE TITLE
Doesn't detect this as a styled call

### ISSUE THREAD

**Issue by Jack-Works** [styled-components/vscode-styled-components#266]
**Describe the bug (including copyable syntax)**

Not detected as styled css call

**Screenshot**
![image](https://user-images.githubusercontent.com/5390719/106109548-a4990080-6184-11eb-9141-a04f11332eda.png)

**To Reproduce**
```js
const Container = styled(ListItem)(
    ({ theme }) => `
    border: 1px solid #f3f3f4;
    border-radius: 6px;
    /* border-radius: ${theme.shape.borderRadius}; */
`,
)
```

**Additional context**

I'm not using styled-components but `styled` from @material-ui/core https://next.material-ui.com/styles/api/#styled-component-styles-options-component

**Comment by adamerose** [comment]
This fixes it

```jsx
import styled, { css } from "styled-components";

const Container = styled(ListItem)(
    ({ theme }) => css`
    border: 1px solid #f3f3f4;
    border-radius: 6px;
    /* border-radius: ${theme.shape.borderRadius}; */
`,
)```

**Comment by oliviertassinari** [comment]
Duplicate of #395
